### PR TITLE
feat: Hack #2 - custom success for form with URL redirect

### PR DESCRIPTION
**Pull request by brianndiiri**

### CODE REVIEW

File: webflow-hacks/form-hacks/2.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import Cookies from 'js-cookie';
```
Comment (alexiglesias93) [inline webflow-hacks/form-hacks/2.ts]: Check what I told @aexshafii in his PR:
https://github.com/finsweet/fs-snippets/pull/7#discussion_r789871917
https://github.com/finsweet/fs-snippets/pull/7#discussion_r789872639

Comment (brianndiiri) [reply webflow-hacks/form-hacks/2.ts]: noted

File: package.json [hunk 2]
```diff
@@ -42,6 +38,7 @@
   },
   "dependencies": {
     "@finsweet/webflow-addons": "^1.0.8",
+    "@types/js-cookie": "^3.0.1",
```
Comment (alexiglesias93) [inline package.json]: This should be a `devDependency`

Comment (brianndiiri) [reply package.json]: done.

**Review by alexiglesias93 — CHANGES_REQUESTED** [review]
You should add the types to `tsconfig.json` so the language server recognizes them.

```json
"types": ["@types/js-cookie"]
```

File: webflow-hacks/form-hacks/2.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  });
+
+  // when the form's submit button is clicked
+  emailForm.addEventListener('submit', function (e) {
+    // if the user selected an option on the select field
+    if (customSuccessMessage.trim() !== '') {
+      // save the customSuccessMessage's value in a cookie
+      Cookies.set('successTextCookie', customSuccessMessage);
+      // then we submit the form
+      return true;
```
Comment (alexiglesias93) [inline webflow-hacks/form-hacks/2.ts]: Same here, no need to return a boolean.

Comment (brianndiiri) [reply webflow-hacks/form-hacks/2.ts]: done

**Comment by brianndiiri** [comment]
@alexiglesias93 this needs a review

**Review by alexiglesias93 — APPROVED** [review]
The Hack #2 looks good, but why does this PR also contain changes for the Hack #1?

**Pull request merged**